src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -65,7 +65,7 @@ public void createFeedbackResponseComments(Collection<FeedbackResponseCommentAtt
     public FeedbackResponseCommentAttributes createEntity(EntityAttributes entityToAdd) 
             throws InvalidParametersException, EntityAlreadyExistsException {
         FeedbackResponseComment createdEntity = (FeedbackResponseComment) super.createEntity(entityToAdd);
-        if (createdEntity == null) {
+        if (createdEntity == null || JDOHelper.isDeleted(createdEntity)) {
```

Comment:
Do we need this check during creation, seems like it will be impossible for isDeleted to be true here?